plone-and-python-community.rst

```diff
@@ -975,24 +975,24 @@ Community
 
     **NO BDFL**
 
-    *15 Years old*
+    *15 years old*
 
-    *The Founders left almost 5 years ago*
+    *The founders left almost 5 years ago*
```

Comment:
I wonder if someone will respond that Alan Runyan is still around and doing Plone